tools/windows-node-installer/pkg/cloudprovider/azure/vm.go

```diff
@@ -577,15 +577,17 @@ func constructAdditionalContent(instanceName, adminUserName, adminPassword strin
 // such as configuring remote management listeners, instance access setup.
 func (az *AzureProvider) constructOSProfile(ctx context.Context) (osProfile *compute.OSProfile, vmName, password string) {
 	instanceName := windowsWorker + randomString(5)
-	adminUserName := "core"
 	adminPassword := randomPasswordString(12)
-	additionalContent := constructAdditionalContent(instanceName, adminUserName, adminPassword)
+	additionalContent := constructAdditionalContent(instanceName, adminPassword)
 	// the data runs the script from the url location, script sets up both HTTP & HTTPS WinRM listeners so that
-	// Ansible can connect to it and run remote scripts on the windows node.
+	// Ansible can connect to it and run remote scripts on the windows node. Also open firewall port 10250.
+
```

Comment:
Don't leave a newline between a comment and what its describing. Put the newline before the comment, not after